src/System.Diagnostics.DiagnosticSource/src/Configurations.props

```diff
@@ -4,7 +4,6 @@
     <BuildConfigurations>
```

Comment:
The only change you need to remove the uap dependency from the package is to change this file to:

```
    <PackageConfigurations>
      netstandard1.3;
      netstandard;
      netcoreapp;
      net45-Windows_NT;      
      net46-Windows_NT;
      netfx-Windows_NT;      
    </PackageConfigurations>
    <BuildConfigurations>
        $(PackageConfigurations);
        uap-Windows_NT;
    </BuildConfigurations>
```

That will still cause uap to build and be consumable in other places like Http but will exclude that configuration from your nuget package.